recipes/xgboost/build.sh

```diff
@@ -0,0 +1,6 @@
+if [ -n "${OSX_ARCH+x}" ]; then
+  patch xgboost/Makefile $RECIPE_DIR/Makefile.patch
+  patch xgboost/build-python.sh $RECIPE_DIR/build-python.patch
+fi
+cp $RECIPE_DIR/LICENSE LICENSE
```

Comment:
I think with the suggested changes I can drop the entire `build.sh` file. I will go ahead and do that.